### PR TITLE
add n23443 logs

### PR DESCRIPTION
**Pull request by glozow**

### CODE REVIEW

**Comment by naumenkogs** [comment]
Thank you very much!

File: _posts/2021-12-08-#23443.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ pr: 23443
 authors: [naumenkogs]
 components: ["p2p"]
 host: naumenkogs
-status: upcoming
+status: past
 commit:
```
Comment (jnewbery) [inline _posts/2021-12-08-#23443.md:10]:
```suggestion
commit: ea3b87af90
```

File: _posts/2021-12-08-#23443.md [hunk 2]
```diff
@@ -66,9 +66,157 @@ commit:
...
+17:10 <gleb7> My first version was 7 commits I think and oh god it was hard to navigate even by myself, we already went through a lot of structural changes since then. Now we're here
+17:10 <jnewbery> I think one downside is that if you're not reviewing the full change set, then you're slightly trusting the author that they're taking you in the right direction
+17:10 <gleb7> Yeah, this is the next question :) 2b Are there any drawbacks to this approach?
+17:11 <gleb7> jnewbery: this is indeed an issue. Not exactly for this PR, where it's a pretty general thing with not so many design choices.
+17:11 <glozow> it might be weird if the sub-PRs don't do much by themselves. lightlike had a good comment about the fact that, if a release was branched off from this PR, nodes would be sending each other meaningless SENDRECONs
+17:11 <gleb7> But I believe the next couple PRs would require more context from future chunks indeed.
+17:12 glozow: I spent some time thinking about this too. I think this is first time we're merging a p2p message which is useless on its own.
```
Comment (jnewbery) [inline _posts/2021-12-08-#23443.md:114]:
```suggestion
17:12 <gleb7> glozow: I spent some time thinking about this too. I think this is first time we're merging a p2p message which is useless on its own.
```

File: _posts/2021-12-08-#23443.md [hunk 3]
```diff
@@ -66,9 +66,157 @@ commit:
...
+17:29 <gleb7> No, this is perfect I think. If you're confused, someone else will also be. And this code should be understandable even without my presence so yeah, I probably should check how good are those comments.
+17:29 <lightlike> stickies-v: I don't think anything is changed in the VERSION message to explicitely indicate recon support.
+17:30 <stickies-v> lightlike oh you're right! 
+17:31 <stickies-v> that also explains the confusion I had about one of your comments regarding peers getting disconnected about sending a SENDRECON message unexpectedly
+17:31 <gleb7> The PR *used to* bump the general Bitcoin p2p protocol version (maybe a month ago), but I dropped this. Does anyone know why?
+17:32 <gleb7> I'm referring to `P2P_VERSION = 70016`
+17:33 <stickies-v> I'm not quite familiar with this, but could it be that an increased p2p protocol version would mean Erlay nodes won't communicate with pre-Erlay nodes anymore, and there's no real need to do that?
+17:33 <lightlike> maybe because it is not really necessary for things to work?
+17:34 when do we change protocol version? when there'd be a compatibility issue?
```
Comment (jnewbery) [inline _posts/2021-12-08-#23443.md:153]:
```suggestion
17:34 <glozow> when do we change protocol version? when there'd be a compatibility issue?
```

**Review by jnewbery — COMMENTED** [review]
Your irc log seems to have dropped the nick on a couple of lines :thinking:

**Comment by glozow** [comment]
Woops, fixed. Thanks @jnewbery

**Pull request merged**